src/github.rs

```diff
@@ -147,24 +147,34 @@ pub trait TryExecute: Executor {
     where
         Self: Sized,
     {
-        #[derive(Deserialize)]
-        struct GithubError {
+        #[derive(Debug, Deserialize)]
+        struct GithubErrorResponse {
             message: String,
+            errors: Option<Vec<GithubError>>,
+        }
+
+        #[derive(Debug, Deserialize)]
+        struct GithubError {
+            resource: String,
+            field: String,
+            code: String,
+            message: Option<String>,
         }
 
         match self.execute::<serde_json::Value>() {
             Ok((_, StatusCode::Ok, Some(response))) => {
                 serde_json::from_value(response).chain_err(|| "Failed to parse response")
             }
             Ok((_, _, Some(response))) => {
-                serde_json::from_value::<GithubError>(response)
+                serde_json::from_value::<GithubErrorResponse>(response)
                     .chain_err(|| "Failed to parse error response")
                     .and_then(|error| Err(error.message.into()))
             }
             Ok((_, _, None)) => Err("Received error response from github with no message".into()),
             Err(err) => Err(err).chain_err(|| "Failed to execute request"),
         }.or_else(|err| {
             error!("Failed to complete request: {}", err);
+            info!("Failed to complete request: {:?}", err);
```

Comment:
maybe this should be `debug!`?

Reply:
That's how I had written it originally, but `debug!` doesn't show up with `-v`. Logging all debug output comes pretty cumbersome and potentially a security concern. It might be worth getting the error parsing down so that it's all shown in the call to `error!`.